phpunit.xml.dist

```diff
@@ -300,8 +300,6 @@
 	<testsuite name="paratest-bigquery">
```

Comment:
teraz som si vsimol, ze to tu zapinas pre BQ ale pre TD je to stale vypnute to testovanie alterTable